german/agot_dynasties/iron_islands_dynasty_names_l_german.yml

```diff
@@ -2,43 +2,43 @@
  #NOTE: follows the order of dynasties in the \common\dynasties\ files
  ### IRON ISLES DYNASTIES/CADET BRANCHES ###
  #Great Houses
- dynn_Hoare:0 "Hoare"
- dynn_Greyjoy:0 "Greyjoy"
- dynn_Greyiron:0 "Greyiron"
- dynn_grey_king:0 "of the Grey King" #to fix localization errors, feel free to come up with a better name.
+ dynn_Hoare:0 "Hoffartt"
+ dynn_Greyjoy:0 "Graufreud"
+ dynn_Greyiron:0 "Graueisen"
+ dynn_grey_king:0 "vom Grauen König" #to fix localization errors, feel free to come up with a better name. #englisch war es "of the Gry King"
  
  #Landed Houses
- dynn_Botley:0 "Botley"
+ dynn_Botley:0 "Botlin"
  dynn_Wynch:0 "Wynch"
- dynn_Stonehouse:0 "Stonehouse"
- dynn_Saltcliffe:0 "Saltcliffe"
+ dynn_Stonehouse:0 "Steinheim"
+ dynn_Saltcliffe:0 "Salzklipp"
  dynn_Merlyn:0 "Merlyn"
  dynn_Myre:0 "Myre"
- dynn_Goodbrother:0 "Goodbrother"
+ dynn_Goodbrother:0 "Guthbruder"
  dynn_Sparr:0 "Sparr"
- dynn_Farwynd:0 "Farwynd"
+ dynn_Farwynd:0 "Weitwynd"
  dynn_Kenning:0 "Kenning"
- dynn_Orkwood:0 "Orkwood"
+ dynn_Orkwood:0 "Orgwald"
  dynn_Sunderly:0 "Sunderly"
- dynn_Blacktyde:0 "Blacktyde"
+ dynn_Blacktyde:0 "Schwarzfluth"
  dynn_Volmark:0 "Volmark"
- dynn_Stonetree:0 "Stonetree"
+ dynn_Stonetree:0 "Steinbaum"
  dynn_Drumm:0 "Drumm"
- dynn_Harlaw:0 "Harlaw"
- dynn_Tawney:0 "Tawney"
- dynn_Codd:0 "Codd"
- dynn_Ironmaker:0 "Ironmaker"
- dynn_Sharp:0 "Sharp"
- dynn_Salt:0 "Salt"
+ dynn_Harlaw:0 "Harlau"
+ dynn_Tawney:0 "Tauny"
+ dynn_Codd:0 "Dorsch"
+ dynn_Ironmaker:0 "Eisenmacher"
+ dynn_Sharp:0 "Scharf"
+ dynn_Salt:0 "Salz"
  
  #Landless Houses
- dynn_Humble:0 "Humble"
- dynn_Shepherd:0 "Shepherd"
- dynn_Netley:0 "Netley"
+ dynn_Humble:0 "Demut"
+ dynn_Shepherd:0 "Hirte"
+ dynn_Netley:0 "Netzlin"
  dynn_Weaver:0 "Weaver"
```

Comment:
Das ist Haus Weber in deutsch
https://eisundfeuer.fandom.com/de/wiki/Haus_Weber_(Eiseninseln)